pyfar/io/io.py

```diff
@@ -501,12 +502,16 @@ def audio_formats():
     return soundfile.available_formats()
 
 
-def audio_subtypes(format=None):
+@pf._utils.rename_arg(
+        {"format" : "audio_format"},
+        "'format' will be deprecated in "
+        "pyfar 0.9.0 in favor of 'audio_format'")
+def audio_subtypes(audio_format=None):
```

Comment:
I'm on favour of changing this, but we should also a test if this raises an error for pyfar>=0.9.0. There should be examples in test_deprecations for other cases.